pkg/clusteragent/admission/mutate/autoinstrumentation/auto_instrumentation.go

```diff
@@ -227,26 +226,6 @@ func profilingClientLibraryConfigMutators(config *webhookConfig) []podMutator {
 	return podMutators
 }
 
-func injectApmTelemetryConfig(pod *corev1.Pod) {
```

Comment:
question: why did you move this from a function call to an inline implementation?

Reply:
The inline implementation is in a `podMutator` which is basically exactly what this is. It didn't seem necessary to keep this around.

Reply:
I could have just called this function there I guess. 🤷

Reply:
@adel121 updated to use the function.

Reply:
Thanks 👍